software/mainboard.py

```diff
@@ -76,16 +104,22 @@ def calculate_throw_strength(self, distance):
         if distance == 0:
             return 0
         else:
-            # int(distance*0.277 + 413)
-            # return int(distance*0.275 + 411)
-            return int(distance*0.270 + 425)
+            #print (int((distance*self.active_slope + self.active_constant)))
+            return int((distance*self.active_slope + self.active_constant))
+            #if self.driving_forward:
+            #    return int(distance * 0.195 + 3589)
+            #else:
+            #    return int(distance * 0.207 + 3625)

```

Comment:
If you are not using the code anymore, remove it instead of commenting it out. Git has version management, you can always go back to look for the old code if you wish so. But commented out code tends to accumulate over time and makes the readability considerably worse